target_postgres/__init__.py

```diff
@@ -145,7 +145,7 @@ def main(config, input_stream=None):
             host=config.get('postgres_host', 'localhost'),
             port=config.get('postgres_port', 5432),
             dbname=config.get('postgres_database'),
-            user=config.get('postgres_username'),
+            user=config.get('postgres_username', 'postgres'),
```

Comment:
This defaults to the system username in unix systems. Idk if we want to override that with postgres